### PR TITLE
Small UX Fixes

### PR DESCRIPTION
**Pull request by geenen124**
- Fix scrolling bar scroll
- Hide scrolling bar when project isn't live
- Fix alignment of live elements in header bar

### CODE REVIEW

File: elements/timeline-view/scrolling-box.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
             }
         </style>
 
-        <div id="scrollingline">
+        <div id="scrollingline" hidden$="{{ !live }}">
```
Comment (martijnjanssen) [inline elements/timeline-view/scrolling-box.html:30]: Spaces are not needed but whatever.

File: elements/timeline-view/scrolling-box.html [hunk 2]
```diff
@@ -72,16 +76,27 @@
                     duration: 0,
                     easing: "linear",
                 });
+
+            const drawer = document.getElementsByTagName("paper-drawer-panel");
+            let containerEl = drawer[0];
```
Comment (martijnjanssen) [inline elements/timeline-view/scrolling-box.html]: Please write full variable names. We had trouble in the past reading variables in your code.

File: elements/timeline-view/scrolling-box.html [hunk 3]
```diff
@@ -72,16 +76,27 @@
...
+
             // eslint-disable-next-line no-undef
             Velocity(scrollLine, // eslint-disable-line new-cap
                 "scroll",
                 {
+                    container: containerEl,
```
Comment (martijnjanssen) [inline elements/timeline-view/scrolling-box.html]: Is this a mexican container? El container?

File: elements/timeline-view/scrolling-box.html (hunk 1 above)
Comment (Markvanderuit) [inline elements/timeline-view/scrolling-box.html:30]: More readable to me with the spaces, so ehh, 'whatever' pretty much sums it up.

**Comment by mennoo1996** [comment]
Nothing to add on Martijn's comments

File: elements/timeline-view/scrolling-box.html (hunk 2 above)
Comment (Markvanderuit) [inline elements/timeline-view/scrolling-box.html]: El containeros needs a longer name, this isn't the first time we've run into this.

**Comment by Markvanderuit** [comment]
Voy fusión esto ahora!

**Pull request merged**